app13.js

```diff
@@ -0,0 +1,19 @@
+console.log("=======================");
+console.log("現在持っているタスク一覧");
+console.log("=======================");
+const housework = ['掃除','買い物','散歩'];
+for ( i = 0; i < housework.length; i++){
+    console.log(length++ + ": " + housework[i] );
+}
```

Comment:
- こちらは単純 for 文ですが 16 行目は `forEach` メソッドになっていて同じファイル内で統一されていないようです。どちらかに統一しましょう
- `forEach` メソッドを使うならアロー関数とのセットで使った方がイマドキの書き方です
- `forEach` メソッドで添え字（インデックス）が利用したい場合は以下の通りにすると利用できますよ。
- テンプレート文字列を使うと文字列処理が簡潔に書けますよ
  - https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Template_literals
  - 消し忘れと思われる `+` がありますので、今後は気を付けてほしいです

```suggestion
housework.forEach((task, index) => {
    console.log(`${index} : ${task}`);
});
```